csharp/protos/unittest_issues.proto

```diff
@@ -116,4 +116,11 @@ message TestJsonFieldOrdering {
     string o2_string = 3;
   }
   
-}
\ No newline at end of file
+}
+
+message TestJsonName {
+  // json_name field options are not properly handled during deserialization
```

Comment:
I'd remove this comment, or change it to a message-level comment of "Message for testing the effects of the json_name option"